setup.py

```diff
@@ -55,7 +55,6 @@ def package_data(pkg, roots):
     ],
     entry_points={
         'xblock.v1': [
-            'flow-control = flow_control:FlowControlXblock',
             'check-point = flow_control:FlowCheckPointXblock',
```

Comment:
at best, we should leave the flow-control name, instead of check-point. That will probably break devstack for a sec, but otherwise the name will always be wrong